src/components/PrimaryPanes/index.js

```diff
@@ -6,6 +6,7 @@
 
 import React, { Component } from "react";
 import { connect } from "react-redux";
+import { Tab, Tabs, TabList, TabPanels } from "react-aria-components/src/tabs";
```

Comment:
We have a somewhat counter intuitive way of vendoring our packages currently:

```diff
diff --git a/src/vendors.js b/src/vendors.js
index 30ee124..8687324 100644
--- a/src/vendors.js
+++ b/src/vendors.js
@@ -23,6 +23,7 @@ import * as devtoolsModules from "devtools-modules";
 import * as devtoolsUtils from "devtools-utils";
 import * as fuzzaldrinPlus from "fuzzaldrin-plus";
 import * as transition from "react-transition-group/Transition";
+import * as reactAriaComponents from "react-aria-components/src/tabs";
 import * as reselect from "reselect";
 import * as url from "url";
 import * as lodashMove from "lodash-move";
@@ -48,6 +49,7 @@ export const vendored = {
   "fuzzaldrin-plus": fuzzaldrinPlus,
   "lodash-move": lodashMove,
   "react-transition-group/Transition": transition,
+  "react-aria-components/src/tabs": reactAriaComponents,
   reselect,
   // Svg is required via relative paths, so the key is not imported path.
   // See .babel/transform-mc.js
```

the rationale is that this lets us separate the files we land directly in firefox repo and the vendor bundle we need to keep separate. Basically src & dist [here][h]

[h]: https://github.com/mozilla/gecko-dev/tree/master/devtools/client/debugger/new

Reply:
I think this is why the integration tests are failing

Reply:
Gotcha. Thanks for the info!